src/cache/tlru-in-memory-assignment-cache.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { TLRUInMemoryAssignmentCache } from './tlru-in-memory-assignment-cache';
+
+describe('ExpiringLRUInMemoryAssignmentCache', () => {
+  let cache: TLRUInMemoryAssignmentCache;
+  const defaultTimout = 60_000; // 10 minutes
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+    cache = new TLRUInMemoryAssignmentCache(2);
+  });
+
+  it(`assignment cache's timeout should default to 10 minutes `, () => {
+    const key1 = { subjectKey: 'a', flagKey: 'b', banditKey: 'c', actionKey: 'd' };
```

Comment:
nit: this is not a cache key because only `subjectKey` and `flagKey` are used as a cache key (see `assignmentCacheKeyToString`) and `banditKey` and `actionKey` are used for cache value. (Got me tripped for a minute because I thought `banditKey`/`actionKey` were part of cache key and that would be a bug.)

A better name for this thing is "cache entry."